<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArmLoginWidget</class>
 <widget class="QFrame" name="CArmLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #21272A, stop:1 #373C3F);
}
</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>256</width>
       <height>420</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>420</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>17</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="uegLabel">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>17</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>54</width>
             <height>17</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
background-image: url(:/CArmWorkStation/ImageSource/Login/Logo.png);
background-position:center left;
background-repeat: no-repeat;
background-color: transparent;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="indent">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="uegLabel_3">
          <property name="minimumSize">
           <size>
            <width>182</width>
            <height>17</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>182</width>
            <height>58</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>182</width>
            <height>58</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel
{
background-image: url(:/CArmWorkStation/ImageSource/Login/C-Arm .png);
background-position:center left;
background-repeat: no-repeat;
background-color: transparent;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>58</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>58</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">QLabel
{
background-color: transparent;
font: 14px &quot;Microsoft YaHei&quot;;
color:rgba(255,255,255,0.4);
border:0px;
}</string>
           </property>
           <property name="text">
            <string>版本号：1.0.0.1</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>251</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>251</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
background-image: url(:/CArmWorkStation/ImageSource/Login/machine.png);
background-position:center left;
background-repeat: no-repeat;
background-color: transparent;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightWidget_2" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: #0E1417;</string>
     </property>
     <layout class="QVBoxLayout" name="rightWidget">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>68</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>66</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>91</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">QLabel
{
background-color: transparent;
font: 24px &quot;Microsoft YaHei&quot;;
color:rgb(255,255,255);
border:0px;
}</string>
        </property>
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">QLabel
{
background-color: transparent;
font: 16px &quot;Microsoft YaHei&quot;;
color:#B4C4D9;
border:0px;
}</string>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border: 1px solid #4a525a;
border-radius:2px;
background:#282f35;
color:#e1e8f1;
font:normal 14px  Microsoft YaHei;
}
QLineEdit:focus
{
border:3px solid #9dadc1;
border-radius:6px
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">QLabel
{
background-color: transparent;
font: 16px &quot;Microsoft YaHei&quot;;
color:#B4C4D9;
border:0px;
}</string>
        </property>
        <property name="text">
         <string>密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pwdEdit">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
border: 1px solid #4a525a;
border-radius:2px;
background:#282f35;
color:#e1e8f1;
font:normal 14px  Microsoft YaHei;
}
QLineEdit:focus
{
border:3px solid #9dadc1;
border-radius:6px
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>54</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="exitBtn">
          <property name="minimumSize">
           <size>
            <width>118</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>118</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #45505C,stop: 1.0 #3A4756);
border:1px solid rgb(74, 82, 90);
border-radius:2px;
color:rgba(255,255,255,0.8);
font: normal 18px  Microsoft YaHei;
}

QPushButton:hover{
color:#ffffff;
}

QPushButton:pressed
{
background-color:#22272C;
color: #AAB3C1;
}

QPushButton:disabled
{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(5,146,252,50),stop: 1.0 rgba(43,174,213,50));
border:1px solid rgb(74, 82, 90, 50);
color:rgb(189, 205, 221, 50);;
}</string>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="submitBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>118</width>
            <height>42</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>118</width>
            <height>42</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(5,146,252,1),stop: 1.0 rgba(43,174,213,1));
border:1px solid rgb(74, 82, 90);
border-radius:2px;
color:rgba(255,255,255,0.8);
font: normal 18px  Microsoft YaHei;
}

QPushButton:hover
{
color:#ffffff;
}

QPushButton:pressed
{
background-color:#22272C;
color: #AAB3C1;
}

QPushButton:disabled
{
background:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgba(5,146,252,50),stop: 1.0 rgba(43,174,213,50));
border:1px solid rgb(74, 82, 90, 50);
color:rgb(189, 205, 221, 50);;
}</string>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="warningLab">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
background-color: transparent;
font: 14px &quot;Microsoft YaHei&quot;;
color: #ff0000;
border:0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>38</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../CArmWorkStation.qrc"/>
 </resources>
 <connections/>
</ui>
